static/sass/_patterns_maas_modal.scss

```diff
@@ -134,6 +134,7 @@ $color-modal-controls: rgba($color-x-dark, .2);
 
   .vbox-next {
     background-image: url('#{$assets-path}9716bbab-chevron.svg');
+    margin-right: 0 !important;
```

Comment:
Please add a comment explaining that it's here to override vanilla button margin.